framework/db/src/com/cloud/utils/db/Transaction.java

```diff
@@ -59,4 +59,30 @@ public T doInTransaction(TransactionStatus status) throws RuntimeException {
         });
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
@jburwell Deprecated methods are all cleaned up from quota code. This particular deprecation is outside quota changes.